ModPatches/Grimworld Angels of Death/Patches/Grimworld Angels of Death/CE_Patch_Turrets.xml

```diff
@@ -2,11 +2,18 @@
 <Patch>
 
 	<Operation Class="PatchOperationReplace">
-		<xpath>Defs/ThingDef[defName="GW_SM_Building_HeavyBolter" or defName="GW_SM_Building_Lascannon"]/fillPercent</xpath>
+		<xpath>Defs/ThingDef[defName="GW_SM_Building_HeavyBolter"]/fillPercent</xpath>
 		<value>
 			<fillPercent>0.85</fillPercent>
 		</value>
 	</Operation>
+	
+	<Operation Class="PatchOperationReplace">
+		<xpath>Defs/ThingDef[defName="GW_SM_Building_Lascannon"]/fillPercent</xpath>
+		<value>
+			<fillPercent>0.50</fillPercent>
```

Comment:
I don't believe this is tall enough to shoot out of embrasures, which players may expect to be able to do. Thoughts?